code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -576,3 +576,10 @@
 	result = "laughter"
 	required_reagents = list("sugar" = 1, "banana" = 1)
 	result_amount = 10
+
+/datum/chemical_reaction/hormone
+	name = "Exotic Hormone"
+	id = "hormone"
+	result = "hormone"
+	required_reagents = list("sugar" = 20, "whiskey" = 20, "teslium" = 20, "mutagen" = 20, "welding_fuel" = 20)
```

Comment:
I don't even...with the advent of Goonchem and more relaistic chemical recipes, why not at least attempt to structure this off of various hormones?